gcli/src/cmd/program.rs

```diff
@@ -60,6 +64,15 @@ pub enum Program {
 }
 
 impl Program {
+    /// Override metadata binary.
+    pub fn override_meta(&self, meta: Vec<u8>) -> Self {
```

Comment:
Maybe we can reflect that we clone `self` in the function name.
```suggestion
    pub fn clone_with_meta_override(&self, meta: Vec<u8>) -> Self {
```

Reply:
makes sense, and this method should be used by users in common cases